src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -142,6 +144,7 @@ export default {
 		MediaDevicesPreview,
 		AppSettingsDialog,
 		AppSettingsSection,
+    CheckboxRadioSwitch,
```

Comment:
Expected indentation of 2 tabs but found 4 spaces

Reply:
Fixed it.

Reply:
But you forgot to push?